### ISSUE TITLE
A11y_AzureDataStudio_BigDataClusterOverviewPage _ScreenReader : NVDA is unnecessarily announcing clickable for every read only data.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7662]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_ADS_OctTestPass;#A11yMAS;#A11yTCS;#MAS1.3.1;DesktopApp;#NVDA;#SQL AzureDataStudio;#Win32

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
NVDA Version: 2019.2

**Additional Details:**
Tool Name : NVDA

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard
8. "Big Data Cluster Overview page " should open.
9. Start NVDA.
10. Navigate till "Cluster Properties" using basic navigation that is using arrow.
11. Observe the announcement by screen reader .

**Actual:**
Screen reader  is unnecessarily announcing clickable for every read only data.
It is announcing like "Clickable Cluster properties".
**Similar issue is also observed with following screens:
1)Control tab**
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard->click on the "control tab " in the left nav bar-> navigate to the "Metricsdc" tab.

**2)Jupyter books**
1. Install and Launch the application the azure data studio application.
2.Connet to the servers with the required credentials.
3.Navigate to the control panel and click on the "jupyter books" icon.
4.Press CRTL+SHIFT+P and search "jupyter books: SQL Server 2019 Guide" 

 
**Expected:**
Screen reader should not announce clickable for the text as it is read only data.

**User Impact:**
Screen reader user will get incorrect information regarding the content present on the screen.

**Attachment for Reference:**
[7662_A11y_AzureDataStudio_BigDataClusterOverviewPage _ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3716321/7662_A11y_AzureDataStudio_BigDataClusterOverviewPage._ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
This was fixed by https://github.com/microsoft/azuredatastudio/pull/8384

**Comment by 12-shweta** [comment]
Verified the bug in the latest version :
Version: 1.15.0-insider (user setup)
Commit: 56695be14ac0160e7a4d23b7af939b98fdaaea45
Date: 2020-01-30T05:35:45.707Z
VS Code: 1.42.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is still reproducing .
Please find the attachment :
![image](https://user-images.githubusercontent.com/54656883/73432836-8c23b980-4369-11ea-8787-02d5513de51c.png)

**Comment by vinodbk197** [comment]
@Charles-Gagnon 
Verified this issue on :Version: 1.15.0-insider (user setup)
The issue is still reproduced, Please refer the above comments of @12-shweta. we are not able to reopen the bug. Kindly reactivate the bug as this bug is still reproduced.

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;
